app/views/layouts/application.html.erb

```diff
@@ -11,15 +11,6 @@
   </head>
 
   <body>
-    <% if signed_in? %>
-      Signed in as: <%= current_user.email %>
-      <%= button_to 'Sign out', sign_out_path, method: :delete %>
-      <image src="<%= (url_for(current_user.avatar)) %>" style="width: 120px;">
-    <% else %>
-      <%= link_to 'Sign in', sign_in_path %>
-    <% end %>
-
-
```

Comment:
But now, how would I get access to the sign-in page? 🤷🏽

Reply:
the sign_in page is set by default in the root with and conditional in the index page 🚀